addon/package.json

```
@@ -69,7 +69,7 @@
     "rollup-plugin-delete": "^2.0.0",
     "rollup-plugin-ts": "^3.2.0",
     "tslib": "^2.4.0",
-    "typescript": "4.9.4"
+    "typescript": "5.6.3"
```

I bet this meant to be
```suggestion
    "typescript": "~5.6.3"
```